source/databases-collections.txt

```diff
@@ -0,0 +1,275 @@
+MongoDB Server manual.
+
+Access a Database
+-----------------
+
+Access a database by using dictionary-style access on your ``MongoClient`` instance.
+
+The following example accesses a database named "test_database":
+
+.. code-block:: python
+
+   database = client["test_database"]
+
+Access a Collection
+-------------------
+
+Access a collection by using dictionary-style access on an instance of your database.
+
+The following example accesses a collection named "test_collection":
+
+.. code-block:: python
+   :emphasize-lines: 2
+
+   database = client.getDatabase["test_database"]
+   collection = database["test_collection"]
+
+.. tip::
+
+   If the provided collection name does not already exist in the database,
+   MongoDB implicitly creates the collection when you first insert data
+   into that collection.
+
+Create a Collection
+-------------------
+
+Use the ``create_collection()`` method to explicitly create a collection in a
+MongoDB database.
+
+The following example creates a collection called "example_collection":
+
+.. code-block:: python
+   :emphasize-lines: 2
+
+   database = client.getDatabase["test_database"]
+   database.create_collection("example_collection")
+
+You can specify collection options, such as maximum size and document
+validation rules, by passing them in as keyword arguments. To see a full list of
+optional parameters, see the `create_collection() API documentation
+<{+api-root+}pymongo/database.html#pymongo.database.Database.create_collection>`__.
+
+Get a List of Collections
+-------------------------
+
+You can query for a list of collections in a database by calling the
+``list_collections()`` method. The method returns a cursor containing all
+collections in the database and their associated metadata.
+
+The following example calls the ``list_collections()`` method and iterates over
+the cursor to print the results:
+
+.. code-block:: python
+
+   collection_list = database.list_collections()
+
+   for c in collection_list:
+       print(c)
+
+To query for only the names of the collections in the database, call the
+``list_collection_name()`` method as follows:
+
+.. code-block:: python
+
+   collection_list = database.list_collection_names()
+
+   for c in collection_list:
+       print(c)
+
+For more information about iterating over a cursor, see :ref:`pymongo-cursors`. 
+
+Delete a Collection
+-------------------
+
+You can delete a collection from the database using the ``drop_collection()``
+method.
+
+The following example deletes the ``test_collection`` collection:
+
+.. code-block:: python
+
+   collection = database["test_collection"];
+   collection.drop();
+
+.. warning:: Dropping a Collection Deletes All Data in the Collection
+
+   Dropping a collection from your database permanently deletes all
+   documents and all indexes within that collection.
+
+   Only drop collections that contain data that is no longer needed.
+
+Configure Read and Write Operations
```

Comment:
I was originally going to split section up, but it started to seem like a lot of repetition so I went back to having them combined (which I also stole from Node). If you think it would be more clear and worth some repetition though I can split it

Reply:
Probably okay as is. I didn't realize there was only one database operation